README.md

```diff
@@ -355,6 +355,52 @@ auth0
 
 For more info please check our generated [documentation](http://auth0.github.io/react-native-auth0/index.html)
 
+### Bot Protection
+
+If you are using the [Bot Protection](https://auth0.com/docs/anomaly-detection/bot-protection) feature and performing database login/signup via the Authentication API, you need to handle the `requires_verification` error. It indicates that the request was flagged as suspicious and an additional verification step is necessary to log the user in. That verification step is web-based, so you need to use Universal Login to complete it.
+
+```js
+const email = 'support@auth0.com';
+const realm = 'Username-Password-Authentication';
+const scope = 'openid profile';
+
+auth0.auth
+  .passwordRealm({
+    username: email,
+    password: 'secret-password',
+    realm: realm,
+    scope: scope,
+  })
+  .then(console.log)
+  .catch(error => {
+    if (error.name === 'requires_verification') {
+      auth0.webAuth
+        .authorize({
+          connection: realm,
+          scope: scope,
+          login_hint: email, // So the user doesn't have to type it again
+        })
+        .then(console.log)
```

Comment:
Let's not log the tokens, just in case someone copy-pastes this
```suggestion
        .then((credentials) => {
          //logged in!
        })
```

Reply:
Changed in https://github.com/auth0/react-native-auth0/pull/332/commits/1a15df4390904bfb8ac1592320cbac55f1e2dde6.